### PR TITLE
Update to more recent java/sbt/scala

### PR DESCRIPTION
**Pull request by zarsky-broad**
Published manually for now, ~~but added a cloud build file for future use.~~

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,61 +1,22 @@
...
+
+# Setup adapted from https://github.com/hseeberger/scala-sbt/blob/master/debian/Dockerfile
+
+RUN \
+  curl -L -o sbt-$SBT_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb && \
+  apt-get update -q && \
+  apt-get upgrade -qq && \
+  apt-get install -y git && \
```
Comment (davidangb) [inline Dockerfile:7]: hm, if this image includes git, I suggest adding that to the readme!

**Review by davidangb — COMMENTED** [review]
one comment inline; this will need updates to use a different (non-`latest`) tag and repo location.

**Comment by zarsky-broad** [comment]
@davidangb Wrote a build script so that this generates multiple tagged images.

File: Dockerfile [hunk 2]
```diff
@@ -1,61 +1,27 @@
...
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb
+
+ARG SCALA_VERSION
+RUN \
+  mkdir /setup-project && \
+  cd /setup-project && \
+  echo "scalaVersion := \"${SCALA_VERSION}\"" > build.sbt && \
+  sbt sbtVersion scalaVersion && \
+  rm -rf /setup-project
```
Comment (davidangb) [inline Dockerfile:26]: I notice that https://github.com/hseeberger/scala-sbt/blob/master/debian/Dockerfile includes
```
echo "case object Temp" > Temp.scala && \
sbt compile && \
```
does that extra compile step downloadadditional libraries that would be helpful to pre-cache?

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Builds pair a scala version with an sbt version
+SCALA_VERSIONS=(  2.12.11   2.12.12)
+SBT_VERSIONS=(    1.4.7     1.4.6)
+
+# $((x - 1)) is needed because otherwise it actually outputs length, but we need to be zero-indexed
+for i in $(seq 0 $((${#SCALA_VERSIONS[@]} - 1))); do
+  IMAGE_NAME="us.gcr.io/$PROJECT/scala-baseimage:jdk11-${SCALA_VERSIONS[i]}-${SBT_VERSIONS[i]}"
```
Comment (davidangb) [inline build.sh]: where is `$PROJECT` populated from? I see that the cloud build yaml specifies it; does that mean this build.sh can only be run successfully via cloud build?

File: Dockerfile (hunk 2 above)
Comment (zarsky-broad) [reply Dockerfile:26]: I don't think so, but I'll add that in and see if the build size changes!

File: build.sh (hunk 3 above)
Comment (zarsky-broad) [reply build.sh]: I'd just been feeding it in manually, I may remove it altogether once we know the actual publish coordinates, though.

File: README.md [hunk 4]
```diff
@@ -1,92 +1,34 @@
...
+### Available tags
 
-It is recommended that the `Dockerfile` be located in the root of the project source directory.  This way, the `ADD . /cromwell` line below is a good convention for adding the project source code to the Docker container.
+Java 11:
 
-Within the project's source tree are two Docker-related files:
+| Scala   | SBT   | Tag                   |
+|---------|-------|-----------------------|
+| 2.12.11 | 1.4.7 | `jdk11-2.12.11-1.4.7` |
+| 2.12.12 | 1.4.6 | `jdk11-2.12.12-1.4.6` |
```
Comment (davidangb) [inline README.md:24]: is it intentional that the later version of scala uses the earlier version of sbt, and the later version of sbt uses the earlier version of scala?

**Review by davidangb — COMMENTED** [review]
one question, otherwise :+1:

File: README.md (hunk 4 above)
Comment (zarsky-broad) [reply README.md:24]: The pairings match rawls (with the update PR I'm about to publish) and orch.

Comment (davidangb) [reply README.md:24]: hmm  ...  could you also publish a 2.12.12-1.4.7, or even 2.12.13-1.4.7 alongside these, and then separately we'll align versions in rawls and orch?

Comment (zarsky-broad) [reply README.md:24]: I'll add them, sure

**Review by davidangb — APPROVED** [review]

**Review by MatthewBemis — APPROVED** [review]

**Pull request merged**